### ISSUE TITLE
What happened to "Home" option in navigation? 

### ISSUE THREAD

**Issue by chau-intl** [osmandapp/OsmAnd#14927]
### Description
In the past I was able to:

1. Long press on any point on the map
2. Select navigation
3. Click on the "From" or "To" input field
4. And from the dialog which pops up, click on the "Home" button to quick select my designated home. 

The "Home" button is no longer visible:

![IMG_20220725_155609](https://user-images.githubusercontent.com/6311686/180795920-bd46e6e1-2c4f-4893-8dc2-79bfa92c3451.jpg)

Has thus feature been removed? 

In the past couple of years the presence of the "Home" button was flaky. Sometimes it would be available sometimes not. But now its never available. 

### Your Environment
OsmAnd Version: 4.2.7
Android/iOS version: android 11
Device model: Xiaomi 9t pro

**Comment by vshcherb** [comment]
I don't think it was not  possible before. Home was and currently present in Favorites

**Comment by sonora** [comment]
It is not needed, really. You can simply create a Favorite called "_Home" , the underscore will sort it to the beginning of the Favorite list and make it button appear as a button exactly where your "?" button currently shows in your above screenshot.

**Comment by chau-intl** [comment]
> I don't think it was not possible before. Home was and currently present in Favorites

After searching for images of Osmand on google I think you might be right. What I remembered was that favourites was sorted (in some cases at least) so that "Home" and "Work" were the first two entries (with their respective icons).

I think giving them priority in the list is a better solution than having to hack the list using underscores to change the sorting.

Another option would be to take advantage of the vacant space below "Address" next to "My position" and place the two "Home"/"Work" icons there. I think the two assigned destinations deserve some priority in that dialog since they are have a special meaning compared to favourites.
